resources/js/components/BuildSummary.vue

```diff
@@ -1042,7 +1046,7 @@ export default {
 
     addNote: function() {
       this.$axios
-        .post('api/v1/addUserNote.php', {
+        .post('/api/v1/addUserNote.php', {
```

Comment:
Does this need to be prepended with `$baseURL`?

Reply:
It doesn't seem like it.  The other `axios` call in the file doesn't have the `$baseURL` like the other links or references to objects, just the leading slash.

Reply:
I don't have a good way to test this locally, but I believe that you're right: https://github.com/Kitware/CDash/blob/0b6006073e3ac23d940e4d632e6bf22af5fcea5d/resources/js/bootstrap.js#L9